.sample.env

```diff
@@ -1,2 +1,4 @@
 # http://ddollar.github.com/foreman/
 RACK_ENV=development
+RECAPTCHA_SITE_KEY=6Lc6BAAAAAAAAChqRbQZcn_yyyyyyyyyyyyyyyyy
```

Comment:
> Do these keys belong in this file, or should there be a recaptcha-specific config?

We should list here default values for all our env variables IMHO.

> Do we need to store the secret key only in the production environment

No, to use `reCAPTCHA` you need the secret key, even in the dev env. This one is just a development one provided by Google.

We will get a new one and it will stay on our production server in the `.env` file.

Reply:
Awesome, sounds good. Please comment the file accordingly.

Reply:
Actually, for now let's move it out from here, we have no time ATM to put every possible ENV setting here, I will move it to an initializer and make it possible to override the default setting from ENV.

Reply:
https://github.com/berkmancenter/lumendatabase/pull/515/commits/206449e1c91aced3207958e47751811e286c4ea7